tools/osx/crosstool/cc_toolchain_config.bzl

```diff
@@ -55,31 +55,6 @@ def _deterministic_libtool_flags(ctx):
     return []
 
 def _impl(ctx):
-    if (ctx.attr.cpu == "armeabi-v7a"):
```

Comment:
there's some more to cleanup in this file, but I would prefer we land https://github.com/bazelbuild/bazel/pull/13448 first so I don't have to resolve conflicts in that